ballerina/pgp_utils.bal

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2024 WSO2 LLC. (https://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+public type PgpEncryptionOptions record {|
```

Comment:
Add API doc comments to all public types

Reply:
Shall we just rename this to `Options`? 
```ballerina
public isolated function encryptPgp(byte[] plainText, byte[] publicKey, *Options options)
                                       returns byte[]|Error = @java:Method {
```

IMO, the words `pgp` and `encrypt` are redundant as this API name itself provides an understanding that we are going to do a PGP encryption. So it is obvious that these `options` are related to PGP encryption. 

WDYT?

Reply:
Yes, we can reduce the redundancy by renaming it to `Options`. I have addressed it here https://github.com/ballerina-platform/module-ballerina-crypto/pull/555/commits/557f72b5647d4a2c64b3fba9af0312fa47475703